app/build.gradle.kts

```diff
@@ -22,6 +24,26 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+
+    signingConfigs {
+        create("releaseConfig") {
+            keyAlias = getPropertyKey("SIGNED_KEY_ALIAS")
+            keyPassword = getPropertyKey("SIGNED_KEY_PASSWORD")
+            storePassword = getPropertyKey("SIGNED_STORE_PASSWORD")
+            storeFile = file(getPropertyKey("SIGNED_STORE_FILE"))
+        }
+    }
+
+    buildTypes {
+        getByName("release") {
+            signingConfig = signingConfigs.getByName("releaseConfig")
+            isDebuggable = false
+        }
+    }
+}
+
+fun getPropertyKey(propertyKey: String): String {
```

Comment:
🚫 **[ktlint]** <standard:function-expression-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function body should be replaced with body expression
<!-- __reviewdog__:ChA4NjVmM2RiOTI4Y2NmZTgyEgZrdGxpbnQ= -->